### ISSUE TITLE
TypeError: AXL model data must be a dictionary

### ISSUE THREAD

**Issue by regsw** [jonathanelscpt/ciscocucmapi#24]
Hi, 

When trying to execute the thin AXL code in the "Quick Start" 

numplan = axl.sql.query('SELECT * FROM numplan')
print(numplan)

I get: TypeError: AXL model data must be a dictionary

I am not sure how to make the data into a dictionary. Thank you.

**Comment by fatestapestry** [comment]
Yeah, this sounds vaguely familiar.  This is internal - not you.  I'll try to have a look for you over the weekend.  Please consider lib as alpha and not production-ready for making live system changes.
